src/Modules/CalcActiveSkill.lua

```diff
@@ -456,7 +456,7 @@ function calcs.buildActiveSkillModList(env, activeSkill)
 
 	-- Apply gem/quality modifiers from support gems
 	for _, value in ipairs(skillModList:List(activeSkill.skillCfg, "SupportedGemProperty")) do
-		if value.keyword == "active_skill" and activeSkill.activeEffect.gemData then
+		if value.keyword == "active_skill" and activeSkill.activeEffect.gemData and not activeSkill.activeEffect.gemData.tags.support then
```

Comment:
Is this always correct? I am not entirely sure if there are any edge cases here.

Reply:
Gems to test are predator support, bane, arcanist brand, shockwave, summon phantasm and maybe a few others I can't remember / find.

Relevant issues are
https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/2295
https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/2719
https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4684

Reply:
Alright. I tested all of the ones you mentioned except Shockwave (it's not a spell and I did not have Empower). All seem to work as expected.

		In-game		PoB
PREDATOR 	no		no
BANE		yes		yes
ARCANIST BRAND	yes		yes
SHOCKWAVE	??		no (empower)
SUMMON PHANTASM	no		no
IMPENDING DOOM	no		no

Reply:
Ingame shockwave doesn't work so it seems good.